ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -513,6 +513,24 @@ AHU_DFSS_DSP_DX2ZC_ECONZ_EFSS_HTZC:
   - EFSS
   - HTZC
 
+AHU_SFSS_SFVSC_CHWSC_RTM_FDPM2X_ESPC_CO2DFVSC_SSPC2X_CO2M6X_ZTM_VOCM_PMM:
+  description: "Single zone AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS
+  - SFVSC
+  - CHWSC
+  - RTM
+  - FDPM2X
+  - ESPC
+  - CO2DFVSC
```

Comment:
This conflicts with `SSPC2X` below. Does the device control to CO2 levels, or does it control to static pressure?

Reply:
This controls to static Pressure but also monitors CO2 in addition.

Reply:
well, is it supply control then? `CO2DFVSC` is not appropriate, would need a new type called `CO2SFVSC`.

Reply:
you may also need to clarify which zone air co2 sensor is referenced by the setpoint. will this utilize an averaging sensor? a max sensor?

Reply:
Updated CO2EFVSC as exhaust fan speed is controlled by the CO2. Max of zone sensor is used for zone air co2.